app/src/test/java/com/example/kotlincalculator/presenter/CalculatorPresenterTest.kt

```diff
@@ -0,0 +1,237 @@
+package com.example.kotlincalculator.presenter
+
+import com.example.kotlincalculator.CalculatorActivity
+import com.example.kotlincalculator.R
+import com.example.kotlincalculator.mvp.model.CalculatorModel
+import com.example.kotlincalculator.mvp.presenter.CalculatorPresenter
+import com.example.kotlincalculator.mvp.view.CalculatorView
+import com.example.kotlincalculator.utils.EMPTY_STRING
+import com.nhaarman.mockitokotlin2.mock
+import junit.framework.Assert.assertEquals
+import org.junit.Before
+import org.junit.Test
+import org.mockito.Mockito.verify
+import org.mockito.MockitoAnnotations
+import org.mockito.Mockito.`when` as whenever
+
+class CalculatorPresenterTest {
+
+    companion object {
+        private const val ZERO: String = "0"
+        private const val ONE: String = "1"
+        private const val TWO: String = "2"
+        private const val FOUR: String = "4"
+        private const val ELEVEN: String = "11"
+        private const val PLUS: String = "+"
+        private const val MINUS: String = "-"
+        private const val MULTIPLY: String = "*"
+        private const val SHARE: String = "/"
+        private const val POINT: String = "."
+    }
+
+    private val mockModel: CalculatorModel = mock()
+    private val mockView: CalculatorView = mock()
+    private val activity: CalculatorActivity = mock()
+    private var presenter: CalculatorPresenter? = null
+
+    @Before
+    fun setup() {
+        MockitoAnnotations.initMocks(this)
+        whenever(mockView.activity).thenReturn(activity)
+        presenter = CalculatorPresenter(mockView, mockModel)
+    }
+
+    @Test
+    fun onNumberPressedWithFirstOperandEmpty() {
+        whenever(mockModel.firstOperand).thenReturn(EMPTY_STRING)
+        whenever(mockModel.operator).thenReturn(EMPTY_STRING)
+        presenter?.onNumberPressed(ONE)
+        verify(mockModel).firstOperand =
```

Comment:
verify method is for verifying public methods (and expected arguments) to verify values actually you need to assert, for example assertEquals(expected, actual)

Reply:
done